### ISSUE TITLE
Tab key stops working when NVDA browses into a <webview>

### ISSUE THREAD

**Issue by juliekoubova** [electron/electron#25277]
### Preflight Checklist
* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details
* **Electron Version:** 10.1.1
* **Operating System:** Windows 10 2004 (19041.450), macOS 10.15.6
* **Last Known Working Electron version:** N/A

### Expected Behavior
Narrated element inside `<webview>` gets correctly focused and `Tab` key keeps working.

### Actual Behavior
Element isn't focused and `Tab`  stops working until focus is moved by mouse or `Shift`+`Tab`.

### To Reproduce
1. Load Fiddle https://gist.github.com/juliekoubova/8644970aa798ddfadf57a6ada1e6c67b
1. Download and run NVDA 2020.2 from https://www.nvaccess.org/download/
1. Make sure you are in NVDA **Browse Mode**. You can toggle it using `Insert`+`Space`. 
    - When in **Browse Mode**, you can use `Up` and `Down` arrows to move virtual focus and NVDA narrates its position.
    - I'm running Windows in Parallels and I used [SharpKeys](https://github.com/randyrants/sharpkeys/releases/tag/v3.9.2) to map my F16 key to Insert.
1. Use the `Down` key to move the virtual focus to the second input box.
1. Press `Tab` and verify that the real focus moves to the second input box, where the virtual focus was.
1. Press `Insert`+`Space` to return to the **Browse Mode**.
1. Use the `Down` key to move the virtual focus into the `<webview>`. Verify that NVDA narrates "*heading level 1 Example Domain*".
1. Press `Tab`
    - The next focusable element (*More Information* link) scrolls into view but is not focused.
    - You cannot use `Tab` to move the focus anywhere now.
    - `Shift`+`Tab` unsticks the focus and makes `Tab` work again.

**Comment by juliekoubova** [comment]
The issue can also be reproduced by pressing `Insert`+`F7` and navigating into the `<webview>` from there.

**Comment by juliekoubova** [comment]
A similar issue also repro's on **macOS 10.15.6** with VoiceOver.
1. Press `Command`+`F5` to enable VoiceOver
1. Press `Control`+`Option`+`U` to show the list of landmarks.
1. Press `Right` arrow until you get the list of links.
1. Select the **SharpKeys** link, press `Enter` and verify that it is focused.
    - Verify that pressing `Tab` moves the focus to the following element, the first input field.
1. Press `Control`+`Option`+`U` to show the list of links.
1. Select the **More Information** link in the `<webview>`.
    - This time, the link is scrolled into view, but is not focused.
1. Press `Tab`
    - Focus moves to the second input field instead of the https://example.org link, since it never left the first input.

**Comment by juliekoubova** [comment]
BTW, this repros all the way back in Electron 2.0.1.

**Comment by juliekoubova** [comment]
It's been recently fixed in Chromium:

https://bugs.chromium.org/p/chromium/issues/detail?id=1130710
https://chromium-review.googlesource.com/c/chromium/src/+/2470334
https://chromium-review.googlesource.com/c/chromium/src/+/2446789
